unstructured/ingest/main.py

```diff
@@ -172,7 +166,13 @@ def run(self):
     "--remote-url",
     default=None,
     help="Remote fsspec URL formatted as `protocol://dir/path`, it can contain both "
-    "a directory or a single file. Supported protocols are: `s3`, `s3a`, `abfs`, and `az`.",
+    "a directory or a single file. Supported protocols are: `gcs`, `gs`, `s3`, `s3a`, `abfs` "
+    "and `az`.",
+)
+@click.option(
+    "--gcs-token",
+    default=None,
+    help="Token used to access Google Cloud. By default shouldn't be needed.",
```

Comment:
rather than "By default shouldn't be needed" might drop more info that you had in comment below. around using default gcloud credentials - attempt to get credentials from the google metadata service - fall back to anonymous access.

Reply:
Good point.